src/platform/screen.c

```diff
@@ -1,22 +1,78 @@
 #include "platform/screen.h"
 
-#include "SDL.h"
-
 #include "game/settings.h"
 #include "graphics/graphics.h"
 #include "graphics/screen.h"
 
+#include "SDL.h"
+
 static struct {
     SDL_Window *window;
     SDL_Renderer *renderer;
     SDL_Texture *texture;
-} SDL = {0, 0, 0};
```

Comment:
Why did you remove the initialization? If the compiler does not clear the memory (some don't...), this will cause crashes when initializing the texture for the first time.